src/docs/WantToHelp/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Want to Help ?'
+order: 8
+description: 'Discover how you could contribute to Mozaic'
```

Comment:
can not could

Reply:
why ?

Reply:
Découvrez comment vous pourriez contribuer à Mozaic. ça marche aussi non ?